src/components/checkout.summarylist.less

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.cart-summary-list {
+  padding: 0;
+  
+  ul {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 10px;
+  }
+
+  ol {
+    margin-top: 0;
+    margin-bottom: 10px;
+  }
+
+  .cart-applied-promotions {
+    display: inline-block;
+    margin-bottom: 5px;
+  }
+
+  .checkout-tax {
+    .checkout-tax-list {
+      padding-left: 1rem;
+      margin-bottom: 1rem;
+    }
+  }
+
+  *,
+  *:before,
+  *:after {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+  }
```

Comment:
Bootstrap is already setting this for the whole site, you can get rid of this whole part.